test/docs/README.md

```diff
@@ -5,6 +5,7 @@
 | [High-Level Description (HLD) Test Specification](dash-test-HLD.md) | High-level design for the testing of devices which conform to the SONiC-DASH requirements.|  
 | [Dash Test Maturity Stages](dash-test-maturity-stages.md) | Describes a progressive approach to DASH testing.|  
 | [DASH SAI-Thrift Test Workflow](dash-test-workflow-saithrift.md) | DASH test workflow with SAI-thrift. |
+| [DASH SAI-Thrift User Guide](https://github.com/opencomputeproject/SAI/blob/088627dd90c3420daf96d294c661b4a152afb01e/ptf/SAI_PTF_user-guide.md) | DASH SAI Thrift Server User Guide to autogenerate test frame work. |
```

Comment:
I'm not sure the word DASH belongs here since the framework is generic to any SAI. Also, we need to make a note to eventually update the URL to the upstream repo's location, this is a dev branch as part of another PR.

Reply:
All the SAI PTF DASH enhancement code and design documents will be upstreamed to SAI. The User guide is for DASH community to start using it and also as you requested it. We will add this to SAI as well.

Reply:
It's fine for now, we'll address it again when the SAI https://github.com/opencomputeproject/SAI/pull/1512 is merged so we can use the permanent URL. Thanks!